lib/livebook/hubs/dockerfile.ex

```
@@ -169,14 +178,25 @@ defmodule Livebook.Hubs.Dockerfile do
           nil
       end
 
+    environment_variables =
+      if config.environment_variables do
+        envs = config.environment_variables |> Enum.into([]) |> format_envs()
```

So it's deterministic and consistent:

```suggestion
        envs = config.environment_variables |> Enum.sort() |> format_envs()
```